### ISSUE TITLE
Mobilanpassa appen -  UX Design

### ISSUE THREAD

**Issue by overath2** [noQ-sweden/noq#594]
Description
As a user I want to have a clear and easy to use user interface when I use the noQ app on my phone. This story covers design for the mobile App for users: Guest and Volunteer. Volunteer UX should be done first.

Design should be easy to apply for other users of the application too in a later phase, but the Assumption is that the users Host and Caseworker use the application on a desktop computer.

**Acceptance Criteria**

- [x] There is a figma design for mobile screen for all activities for the Guest and Volunteer users (Portrait and landscape mode).
- [x] Navigationsmenyer visas som hamburgermeny på mobila enheter.

Subtasks

Definition of Done
OK from wingmate
Code checked in
All subtasks done or cancelled

**Comment by patrik-pwned** [comment]
Känns angeläget nu när vi skapar volontärer som användare om de ska kunna boka övernattningar åt gäster. Något nytt kring detta? @overath2 eller @callekrook

**Comment by overath2** [comment]
Pass

**Comment by overath2** [comment]
@ChristofferEriksson-NoQ hinner du lämna över denna eller bli klar innan du slutar?

**Comment by LudvigErlandsson** [comment]
@inhyenoq Hej Inhye, vad behöver du för att kunna skapa design för mobilanpassning för gäst och stödperson? Kontakta mig och @FannyBacklin om du vill få stöd.

**Comment by ChristofferEriksson-NoQ** [comment]
@overath2 Ingen fara! Tänkte göra klart denna innan jag slutar. Jag borde bli klar innan fredag!

**Comment by ChristofferEriksson-NoQ** [comment]
Hello hello! Pardon the delay. Me and Vincent have converted the screens of the existing app into a more mobile friendly option for both portrait and landscape mode.
https://www.figma.com/design/CwWSHrCSf7igChqclirwLp/noQ?node-id=5653-9600&t=krfgobOKusfDlJoT-1
https://www.figma.com/design/CwWSHrCSf7igChqclirwLp/noQ?node-id=5619-7209&t=krfgobOKusfDlJoT-1

We also made two variants with a bit of alternative layout which might be more relevant for the future if we decide to restructure the app a little, these variants have been placed a little ways away from the others to avoid mixups.
https://www.figma.com/design/CwWSHrCSf7igChqclirwLp/noQ?node-id=5619-7210&t=krfgobOKusfDlJoT-1
https://www.figma.com/design/CwWSHrCSf7igChqclirwLp/noQ?node-id=5681-10301&t=krfgobOKusfDlJoT-1

figma designs of the open hamburger menus will follow shortly!

**Comment by overath2** [comment]
Looks awesome. Is the review stuff included?

**Comment by ChristofferEriksson-NoQ** [comment]
We have implemented most of the feedback and annotated some of the specifics we havent addressed yet, such as fully improving call to actions and usage of language. Some of the greater adviced changes were implemented in the alternative layouts as they would require some functions to be moved around on the site.

If you are refering to Johan's UX review then I have included links to the figma with visual annotations both on slack and in the summary document. I'll post links to both below
https://docs.google.com/document/d/1ELJkb4cueQk13UfohGm83nWzWjY2dfSjwaTuw8C0JY8/edit?pli=1&tab=t.0
https://www.figma.com/design/EySrPLK7YyddZMMi0HGHRu/UX-assessment-feedback-29%2F01?node-id=0-1&t=EN4jrfMV6GM4v0ii-1

**Comment by VincentSehlin** [comment]
Today we have have done the second subtask of this ticket which was a navigation bar for the mobile app. It resembles the desktop version. 

Today, we finished the second subtask of this ticket by implementing the navigation bar for the mobile app.
- It mirrors the desktop design to maintain a consistent user experience across platforms.
- We focused on making key navigation items clear and easy to access.
